sdk/core/Azure.Core.Expressions.DataFactory/src/Azure.Core.Expressions.DataFactory.csproj

```
@@ -14,7 +14,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" VersionOverride="1.36.0" />
+    <PackageReference Include="Azure.Core" />
+    <PackageReference Include="System.ClientModel" />
```

```suggestion
    <PackageReference Include="System.ClientModel" />
```

This reference comes in transitively via Azure.Core.   Please don't add an additional direct.

---

True

---

removed